packages/lesswrong/lib/collections/users/views.ts

```diff
@@ -27,7 +27,8 @@ declare global {
 
 // Auto-generated indexes from production
 ensureIndex(Users, {username:1}, {unique:true,sparse:1});
-ensureIndex(Users, {"emails.address":1}, {unique:true,sparse:1});
+ensureIndex(Users, {email:1}, {unique:true,sparse:1});
```

Comment:
I believe this index creation will fail unless the `email` collection already has everything unique

Reply:
Ah, it most definitely does not